.travis.yml

```diff
@@ -1,10 +1,19 @@
 language: csharp
-mono:
-  - 4.8.1
+sudo: required
+dist: trusty
+dotnet: 1.0.0-preview2-003121
 solution: GoogleMapsApi.sln
+
+addons:
```

Comment:
There are two kinds of support:

Mono (which net45 and below) and Dotnet core, which is supported out of box for preview 1 which is on the old xproj files I believe, which is prior to vs 2017 where they became MS build again on csproj files. I think its wise to skip xproj and go straight to csproj in my experience.